skyportal/tests/frontend/sources_and_followup_etc/test_followup_requests.py

```diff
@@ -52,6 +52,16 @@
     ztf_isonline = True
 
 
+swift_url = "https://www.swift.psu.edu/toop/submit_json.php"
```

Comment:
Do we need to test against swift if we have the pre-recorded transactions?

Reply:
We do the others...